### DISCUSSION TITLE
Which bugs are fixed where?

### DISCUSSION THREAD

**Discussion by tmechsner** [saleor/saleor#10000] [General] [answered]
I just saw that "invalid undiscounted total on order line" has been fixed on 3.3 and 3.4, but not on 3.2 and 3.1. I have seen similar things a few times - bugs being fixed in one version but not in others. Are those releases deprecated already?! Or what's the rationale behind this?

**Comment by timuric** [comment, marked as answer]
@tmechsner we are only back porting critical bugs and security issues. We aim to introduce a minimal amount of breaking changes in minor releases and advice you to update Saleor as often as possible.
